.github/ISSUE_TEMPLATE/custom-pull-request-template.md

```diff
@@ -0,0 +1,30 @@
+# Description
+
+Please include a summary of the changes and attch the related issue link. 
+
+Summary: 
+
+Issue Link: 
+
+# Date
+
+2023/9/18 21:22pm
+
+# Type of action
+
+- [ ] Bug fix 
+- [ ] New feature
+- [ ] Requires a documentation update
+
+
+# Checklist:
+
+- [ ] My code style is consistent with the current project
+- [ ] I have done a brief review of my code
+- [ ] I have written readable comments
+- [ ] I have updated the documentation which related to the changes I made
+- [ ] I have written reasonable tests and the updated version all passed
+- [ ] There should no potential errors occur after the changes
+
+# Action Required
+Please mention the further step need to do if required.
```

Comment:
Could we rephrase this to `Please mention the further steps needed to do if necessary.`, thanks!